### PR TITLE
fix list.LengthMismatch not in stdlib anymore

### PR DESCRIPTION
**Pull request by inoas**
I assume you are OK with Error(Nil) here as a return value from the function.

### CODE REVIEW

**Comment by inoas** [comment]
strictly speaking if we returned `Error(list.LengthMismatch)` before and return `Error(Nil)` now, that is a breaking change and non_empty_list should go to 2.0?

If you agree, let me now I fix the PR.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## v1.1.2 - 2024-02-10
```
Comment (giacomocavalieri) [inline CHANGELOG.md]: Could you not update the package version and call this "## Unreleased"

File: gleam.toml [hunk 2]
```diff
@@ -1,13 +1,14 @@
 name = "non_empty_list"
-version = "1.1.1"
+version = "1.1.2"
```
Comment (giacomocavalieri) [inline gleam.toml]: Could you revert the version to `1.1.1`?

File: src/non_empty_list.gleam [hunk 3]
```diff
@@ -1,9 +1,14 @@
+//// Non-empty lists in Gleam
+////
+//// lists that is guaranteed to contain at least one item.
```
Comment (giacomocavalieri) [inline src/non_empty_list.gleam]: Could you remove this?

**Review by giacomocavalieri — COMMENTED** [review]
Thank you so much! I've left just a couple of comments.
> strictly speaking if we returned Error(list.LengthMismatch) before
> and return Error(Nil) now, that is a breaking change and non_empty_list
> should go to 2.0?

Yeah that's it, but don't worry about the version change, I'll deal with that when publishing a new one

**Comment by giacomocavalieri** [comment]
Thank you!

**Pull request merged**